.github/workflows/CI.yml

```diff
@@ -26,16 +26,19 @@ jobs:
           - os: ubuntu-latest
             version: 1.0.5
             arch: x64
+          # Add a 1.3 job because that's what Invenia actually uses
+          - os: ubuntu-latest
+            version: 1.3
+            arch: x64
           # Add a nightly job just to check if anything upcoming is breaking
           # Unfortunately, this may create some noise in pull requests
           - os: ubuntu-latest
             version: nightly
             arch: x64
           # Add a 32-bit job to ensure we don't have any 64-bit specific logic
-          # Currently disabled because of issue #80
-          # - os: ubuntu-16.04
-          #   version: 1.3
-          #   arch: x86
+          - os: ubuntu-16.04
```

Comment:
Is this the last version of ubuntu that supports 32-bit?

Reply:
I don't know for github actions. I know that's the only version I could get for a local vm. Anyways, I tried changing it and if that doesn't work I'll reverse it.

Reply:
Looks like that works 👍